gradle.properties

```diff
@@ -2,11 +2,11 @@ ideaVersion=IC-2019.3
 # build number ranges
 # https://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html
 sinceIdeaBuild=193
-projectVersion=0.2.3
+projectVersion=0.2.4
```

Comment:
Why do we need this bump ?

Reply:
0.2.3 was merged but had intellij-common 1.3.0-SNAPSHOT.

Reply:
Yes but not published. Anyway, final decision is yours